R/initialize_modules.R

```diff
@@ -74,45 +74,48 @@ initialize_module <- function(parameters, data, module_name) {
 
   if (module_class_name == "Fleet") {
     module_fields <- setdiff(module_fields, c(
-      "log_expected_index",
-      "proportion_catch_numbers_at_age"
+      "log_index_expected",
+      "log_landings_expected",
+      "composition_numbers_at_age",
+      "proportion_numbers_at_age",
+      "observed_index_in_weight",
```

Comment:
if indices are only allowed to be in biomass/weight. That would exclude many/all assessments in the NEUS.

Reply:
They were only allowed in weight before. Now this and the landings equivalent code allow either to be in weight or numbers.